components/desktop/mate/caja/caja.p5m

```diff
@@ -247,11 +247,35 @@ file path=usr/share/man/man1/caja-autorun-software.1
 file path=usr/share/man/man1/caja-connect-server.1
 file path=usr/share/man/man1/caja-file-management-properties.1
 file path=usr/share/man/man1/caja.1
+#file path=usr/share/mime/XMLnamespaces
```

Comment:
They were not in the manifest but appeared when I generate it... no idea if they should be in there.